spec/spec.md

```diff
@@ -1249,6 +1253,8 @@ format-related rules above:
               "enum": ["all", "pick"]
             },
             "count": { "type": "integer", "minimum": 1 },
+            "min": { "type": "integer", "minimum": 0 },
```

Comment:
Schema was missing properties. (the overall Presentation Definition Schema had those...)